client/components/domains/registrant-extra-info/ca-form.jsx

```diff
@@ -155,7 +155,7 @@ class RegistrantExtraInfoCaForm extends React.PureComponent {
 					<FormLabel>
 						<FormCheckbox
 							id="cira-agreement-accepted"
-							defaultChecked={ true }
+							checked={ ciraAgreementAccepted }
```

Comment:
Yeah, I'm ok with having it checked by default if that's what we do want to do.

The defaultChecked got weird when I cancelled the domain and started another one. I got back to this dialog with the box ticked along with the warning and the button disabled, and then when I clicked on it, it unticked, but the warning and disabled button remained, until I clicked it again.